code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PushTemplateConstants.kt

```diff
@@ -53,7 +46,12 @@ internal object PushTemplateConstants {
     }
 
     internal object DefaultValues {
-        const val SILENT_NOTIFICATION_CHANNEL_ID = "AEPSDK Silent Push Notifications"
+        // When no channel name is received from the push notification, this default channel name is used.
+        // This will appear in the notification settings for the app.
+        const val DEFAULT_CHANNEL_NAME = "AEPSDK Push Notifications"
```

Comment:
Out of the scope of this PR: Can we check with PM if these are the final default channel names we want to use? Since these names shows up in the notification settings for the app, it would be better to have something generic like "Default"

Reply:
i can check with PM about these names